### PR TITLE
New graph

### PR DESCRIPTION
**Pull request by kkardatos**

### CODE REVIEW

File: front-end-vue/src/helpers/GraphTranslator.ts [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+function addNodes(entity: any, keys: string[], firstNode: TTGraphData, predicates: any): void {
+  if (isObjectHasKeys(entity)) {
+    keys.forEach(key => {
+      if (isArrayHasLength(entity[key]) && key === SHACL.PROPERTY) {
+        entity[key].forEach((nested: any) => {
+          firstNode.children.push({
+            name:
+              nested?.[SHACL.CLASS]?.name ||
+              getNameFromIri(nested?.[SHACL.CLASS]?.["@id"]) ||
```
Comment (matt-voror) [inline front-end-vue/src/helpers/GraphTranslator.ts:33]: Refactor to reduce inline if's ('?') and inline OR's ('||').

File: front-end-vue/src/helpers/GraphTranslator.ts [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+export function translateFromTTDocument(ttdocument: any): TTGraphData {
+  ttdocument["@context"];
+  const firstNode = {
+    name: ttdocument.entities[0]["rdfs:label"],
+    iri: ttdocument.entities[0]["@id"],
+    relToParent: "",
+    children: [],
+    _children: []
+  } as TTGraphData;
+  const excludedPredicates = [IM.MATCHED_TO, IM.HAS_STATUS, IM.STATUS, RDFS.COMMENT, RDF.TYPE, RDFS.LABEL];
```
Comment (matt-voror) [inline front-end-vue/src/helpers/GraphTranslator.ts:64]: extract excludedPredicates into translator wide static variable as used in multiple places

File: front-end-vue/src/helpers/GraphTranslator.ts [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+          firstNode.children.push({
+            name:
+              nested?.[SHACL.CLASS]?.name ||
+              getNameFromIri(nested?.[SHACL.CLASS]?.["@id"]) ||
+              nested?.[SHACL.DATATYPE]?.name ||
+              getNameFromIri(nested?.[SHACL.DATATYPE]?.["@id"]),
+            iri: nested?.[SHACL.CLASS]?.["@id"] || nested?.[SHACL.DATATYPE]?.["@id"],
+            relToParent: nested[SHACL.PATH].name,
+            children: [],
+            _children: []
```
Comment (matt-voror) [inline front-end-vue/src/helpers/GraphTranslator.ts:39]: how is children different from _children? Consider renaming _children to be more specific

File: front-end-vue/src/views/Concept.vue [hunk 4]
```diff
@@ -79,6 +79,11 @@
                 <Members :conceptIri="conceptIri" @memberClick="active = 0" />
               </div>
             </TabPanel>
+            <TabPanel header="TTGraph">
```
Comment (matt-voror) [inline front-end-vue/src/views/Concept.vue:82]: TTGraph rename -> Interactive graph? Visualisation?

File: front-end-vue/src/helpers/GraphTranslator.ts (hunk 3 above)
Comment (kkardatos) [reply front-end-vue/src/helpers/GraphTranslator.ts:61]: _children is just a copy of children and they are exactly the same. it is used in order to keep a reference to the children of a node for the toggle functionality. _variable in general is a convention for naming "private" or unused properties/methods in js

**Pull request merged**